thesis-gui/src/context/CategoryContext.tsx

```diff
@@ -38,6 +38,8 @@ export const CategoryContextProvider: FC = ({ children }) => {
             getCategories();
         } else {
             setChosenCategories([]);
+        } else {
```

Comment:
double **else**

Reply:
A, yes, it happens after merge. I will fix it